client/dashboard/me/security-password/summary.tsx

```diff
@@ -1,15 +1,29 @@
+import { userSettingsQuery } from '@automattic/api-queries';
+import { useSuspenseQuery } from '@tanstack/react-query';
 import { Icon } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { lockOutline } from '@wordpress/icons';
 import RouterLinkSummaryButton from '../../components/router-link-summary-button';
+import type { SummaryButtonBadgeProps } from '@automattic/components/src/summary-button/types';
 
 export default function SecurityAccountRecoverySummary() {
+	const { data: userSettings } = useSuspenseQuery( userSettingsQuery() );
+
+	const { is_passwordless_user } = userSettings;
+
+	const badges: SummaryButtonBadgeProps[] = [
```

Comment:
Nit: I noticed that the design doesn't show the "Password set" badge for non-passwordless users, and it does seems odd that we show "Password set" given that it's the expected condition for those users.